App/static/js/content-recommendation.js

```diff
@@ -0,0 +1,379 @@
+function recommendBooks() {
```

Comment:
@ZebraAlgebra this function is hooked up to the onClick event of the button, it makes an API call in the `fetch() ...` block to the Route you define in Flask (app.py)